api/prism_app/export_maps.py

```diff
@@ -177,6 +197,7 @@ async def render_single_map(
             {"width": viewport_width, "height": viewport_height}
         )
         await page.emulate_media(media="screen")
+        url = url.replace("localhost:3000", "host.docker.internal:3000")        
```

Comment:
I couldn’t get things to work locally without this, but not sure it’s something we want to merge in…